src/iOS/Avalonia.iOS/Storage/IOSStorageProvider.cs

```diff
@@ -99,6 +100,40 @@ public async Task<IReadOnlyList<IStorageFile>> OpenFilePickerAsync(FilePickerOpe
             ? new IOSStorageFolder(url) : null);
     }
 
+    public Task<IStorageFile?> TryGetFileFromPath(Uri filePath)
+    {
+        // TODO: research if it's possible, maybe with additional permissions.
+        return Task.FromResult<IStorageFile?>(null);
```

Comment:
My knowledge of iOS sandboxing ends here. And I wouldn't say it's safe to just create a new file/folder from the Uri, without checking for the permissions to the file/folder first. 
So instead, I made it to return null just like browser backend.